integrations/jenkins/Jenkinsfile

```diff
@@ -487,7 +487,7 @@ pipeline {
                     /opt/ort/bin/set_gradle_proxy.sh
 
                     rm -fr out/results
-                    /opt/ort/bin/ort $ORT_OPTIONS analyze -i "$PROJECT_DIR/source" -o out/results/analyzer
+                    /opt/ort/bin/ort $ORT_OPTIONS analyze -i "$PROJECT_DIR/source" -o out/results/analyzer -l JENKINS_BUILD_URL=$BUILD_URL
```

Comment:
I wonder if `CI_BUILD_URL` would make sense to use across CI integrations. What do you think?

Reply:
Or `CI_JOB_URL`.

Reply:
I deliberately went for the same name that Jenkins uses plus a `JENKINS_` prefix. I didn't see a need to use a generic name here because the label is not going to be used programmatically or so.